src/prototype_room_init.js

```diff
@@ -286,6 +310,8 @@ Room.prototype.setup = function() {
   });
   let paths_sorted = _.sortBy(paths_controller, sorter);
   let path = this.getMemoryPath(paths_sorted[paths_sorted.length - 1].name);
+  let pathLB = this.getMemoryPath(paths_controller[4].name);
```

Comment:
Why `[4]`.

You can get the mineral path, by getting the id and use `pathStart-ID` path.